CordovaLib/Classes/Public/CDVAppDelegate.m

```diff
@@ -79,8 +79,16 @@ - (BOOL)application:(UIApplication*)application openURL:(NSURL*)url sourceApplic
         return NO;
     }
 
+    NSDictionary *openURLData = @{
+            @"url": url,
+            @"sourceApplication": sourceApplication,
+            @"annotation": annotation
```

Comment:
Good point @pbudzinsky .  I should revert to my original commit where I did explicit checks on sourceApplication and annotation then. sourceApplication can be nil too